webapp/app/components/MessageList/MessageList.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import { Typography } from 'material-ui';
+
+import { Wrapper, Content, Loader } from './MessageList.styles';
+import { Message } from '../Message/Message.component';
+
+
+export class MessageList extends PureComponent {
+  static propTypes = {
+    messages: PropTypes.instanceOf(Map),
+  };
+
+  renderList = () => {
+    if (this.props.messages) {
```

Comment:
if (this.props.messages.size > 0)

oraz dodaj jako defaultProps messages z pustym Map(), unikniesz wtedy dwoch ifow

Reply:
dodałem w selektorze 
`state.get('messages'*, Map()*)`
bo akurat dla `messages`, wychodzi na to samo, mam rację ?

Reply:
Tutaj jeśli mamy domyślnie jakąś wartość pojawia się jeden problem, rozróżnienie kiedy nie mamy żadnych `messages`, od tego kiedy jeszcze się nie załadowały.